tensorflow_probability/python/experimental/stats/sample_stats.py

```diff
@@ -22,15 +22,19 @@
 # Dependency imports
 
 import tensorflow.compat.v2 as tf
-
+from tensorflow_probability.python.internal import nest_util
 from tensorflow_probability.python.internal import prefer_static as ps
+from tensorflow_probability.python.mcmc import diagnostic
+from tensorflow.python.util import nest  # pylint: disable=g-direct-tensorflow-import
 
 
 __all__ = [
-    'RunningMean',
-    'RunningMeanState',
     'RunningCovariance',
     'RunningCovarianceState',
+    'RunningMean',
+    'RunningMeanState',
+    'RunningPotentialScaleReduction',
```

Comment:
Eh... this is kind of MCMC specific. E.g. you can do a RunningCovariance on any dataset, but PotentialScaleReduction is basically only relevant to MCMC chains.

In this case it would have been fine to just implement everything in the reducer. I guess it's fine to leave it as you did it though, the sample_stats API is a bit less assumptive.